### PR TITLE
docs: excalidraw package usage example tweaks

### PR DESCRIPTION
**Pull request by dwelle**
- move `onResize` into `useEffect` (h/t @dai-shi)
- shuffle code around so you don't need to jump around too much when reading the example

### CODE REVIEW

File: src/packages/excalidraw/README.md [hunk 1]
```diff
@@ -111,12 +103,14 @@ export default function App() {
...
+          width={dimensions.width}
+          height={dimensions.height}
           initialData={InitialData}
-          onChange={onChange}
+          onChange={(elements, state) => {
+            console.log("Latest elements:", elements, "Latest state:", state);
+          }}
           user={{ name: "Excalidraw User" }}
-          onPointerUpdate={(payload) => console.log(payload)}
+          onPointerUpdate={(pointerData) => console.log(pointerData)}
```
Comment (ad1992) [inline src/packages/excalidraw/README.md:113]:
```suggestion
          onPointerUpdate={(pointerData) => console.log(pointerData)}
          onCollabButtonClick={() => window.alert("You clicked on collab button")}
```

**Review by ad1992 — APPROVED** [review]
can you also update the changelog?
Or we can add a check to skip changelog for changes in the readme as that also should be fine

**Comment by dwelle** [comment]
@ad1992 I've refactored the changelog-check and added readmes to the ignore. Can you please double-check everything looks ok?

File: scripts/changelog-check.js [hunk 2]
```diff
@@ -1,34 +1,34 @@
...
-        process.exit(0);
-      }
+const IGNORED_PATHS = [
+  "src/excalidraw-app",
+  "packages/utils",
+  "CHANGELOG.md",
+  "README.md",
+].map(normalizePath);
 
-      const onlyNonSrcFilesUpdated = stdout.indexOf("src") < 0;
```
Comment (ad1992) [inline scripts/changelog-check.js:15]: this will help in early return so we should keep this

Comment (dwelle) [reply scripts/changelog-check.js:15]: performance-wise it's totally negligible and I think it's best to colocate the logic as much as possible

Comment (ad1992) [reply scripts/changelog-check.js:15]: true but just feels unnecessary to run the checks for non src files hence suggested keeping it. 
Rest all lgtm. Feel free to merge 👍

**Pull request merged**